pages/dao/[symbol]/params/index.tsx

```diff
@@ -232,252 +232,265 @@ const Params = () => {
                   <Tabs
                     activeTab={activeTab}
                     onChange={(t) => setActiveTab(t)}
-                    tabs={['Params', 'Accounts']}
+                    tabs={['Params', 'Accounts', 'Statistics']}
                   />
                 ) : null}
-                {activeTab === 'Params' ? (
-                  <>
-                    <DisplayField
-                      label="owner"
-                      padding
-                      val={activeGovernance.owner.toBase58()}
-                    />
-                    {realmAccount?.authority?.toBase58() ===
-                      activeGovernance.pubkey.toBase58() && (
-                      <DisplayField
-                        label="Realm Authority"
-                        padding
-                        val={'Yes'}
-                      />
-                    )}
-                    <DisplayField
-                      label="Proposals Count"
-                      padding
-                      val={activeGovernance.account.proposalCount}
-                    />
-                    <DisplayField
-                      label="Voting Proposals Count"
-                      padding
-                      val={activeGovernance.account.votingProposalCount}
-                    />
-                    <DisplayField
-                      label="Max Voting Time"
-                      padding
-                      val={getFormattedStringFromDays(
-                        activeGovernance.account.config.maxVotingTime /
-                          SECS_PER_DAY
-                      )}
-                    />
-                    {communityMint && (
-                      <DisplayField
-                        label="Min community tokens to create a proposal"
-                        padding
-                        val={fmtMintAmount(
-                          mint,
-                          activeGovernance.account.config
-                            .minCommunityTokensToCreateProposal
+                {(() => {
```

Comment:
Hi so now if we have more tabs it would be good to use guard clauses and move it to component.

Reply:
Thanks for the feedback, I'll change them to guard clauses!

By this, "move it to component instand of using function", did you mean I should create separate components for each of the tabs and render them in? Or keep what I did, but using guard clauses instead?

Reply:
Yes it would be good to put it in 3 separate components like AccountsView, paramsView, statsView and use guard clauses to render them.

Reply:
Got it, where would you want me to store the components? With separate .tsx files in the same params folder, or make the components directly within the index.tsx file?

Reply:
Put it in separated files.

Reply:
How's this, I created separate components in a "components" folder within "params". Then used conditionals this way. Unless you prefer I use `if()` statements?
![guardClauseChanges](https://user-images.githubusercontent.com/59552589/160010723-01b941db-c35b-4717-bf80-4e5649f91765.png)

Reply:
Hmm, seems like there are changes being made right now on the params page. I'll just wait for all of them to prevent conflict

Reply:
> How's this, I created separate components in a "components" folder within "params". Then used conditionals this way. Unless you prefer I use `if()` statements? ![guardClauseChanges](https://user-images.githubusercontent.com/59552589/160010723-01b941db-c35b-4717-bf80-4e5649f91765.png)

all good now

> Hmm, seems like there are changes being made right now on the params page. I'll just wait for all of them to prevent conflict

Its finished one small change was needed.

Reply:
Hmm looks like there was some conflict, seems like it's from the DisplayField component. I'll try and fix it, but I was wondering if it might be better to separate DisplayField into it's own component too? 

Since it's used by the `AccountsView` `ParamsView` and `StatsView` components as well

Reply:
Change was to only match program with its known name we can leave display field as its now.

Reply:
I must've messed something up with this branch then. Seems like it's not passing the tests so I'll close this branch/PR and make a new one